pkg/apiserver/controller/queue/queue.go

```diff
@@ -338,26 +338,33 @@ func UpdateQueue(ctx *logger.RequestContext, request *UpdateQueueRequest) (Updat
 
 	// validate fields if not nil, validate namespace at first
 	updateClusterRequired := false
+	resourceUpdated := true
 
 	// validate MaxResource or MinResource
-	scalarResourceLaws := config.GlobalServerConfig.Job.ScalarResourceArray
-	if request.MaxResources.CPU != "" && request.MaxResources.Mem != "" {
-		if err = schema.ValidateResourceInfo(request.MaxResources, scalarResourceLaws); err != nil {
-			ctx.Logging().Errorf("update queue failed. error: %s", err.Error())
+	if request.MinResources.CPU != "" || request.MinResources.Mem != "" {
+		if err := validateQueueResource(request.MaxResources, &queueInfo.MaxResources); err != nil {
```

Comment:
Line 344 checks `request.MinResources`, and Line 345 checks `request.MaxResources`

Reply:
repaired